models/commons/src/reference/generator.ts

```diff
@@ -6,6 +6,7 @@ const generator = Generator<TReference>({
   fields: {
     id: fake((f) => f.datatype.uuid()),
     typeId: null,
+    key: fake((f) => f.lorem.slug(2)),
```

Comment:
We need to be able to configure `key` [like so](https://github.com/commercetools/test-data-research/blob/main/src/entity-drafts/carts.ts#L63)

Reply:
It might - perhaps something consumers should be responsible for? We can be in charge of generating the appropriate fields, but implementation details might be on them 🤷‍♂️

Reply:
I'm thinking about `.random()`, that will generate both in the current configuration.

Reply:
We could leave `key` as `null`, that seems a good tradeoff. We'll have to configure it anyway. I can add a comment to document this as well

Reply:
Resolved in 0cb84b0a178dabfb721bece3bfbcd9f0f73b5e98